vars/artifactSetVersion.groovy

```diff
@@ -1,4 +1,5 @@
 import static com.sap.piper.Prerequisites.checkScript
+import static com.sap.piper.BashUtils.quoteAndEscape as q
```

Comment:
Shouldn't this be contained in the other commit "refactor: use import alias" ?

Reply:
This step was not using quoting at all before, so this change is not about using aliases but about doing quoting.

Reply:
But [here](https://github.com/SAP/jenkins-library/pull/5164/files#diff-2664442056a33e1716e65d35b64f50c555183e6a5dd168ec073034e996abfefaR3).
Anyway, I would not create a separate PR...